connector/connect/common/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -838,6 +841,12 @@ message CoGroupMap {
 
   // (Required) Input user-defined function.
   CommonInlineUserDefinedFunction func = 5;
+
+  // (Optional) Expressions for sorting. Only used by Scala Sorted CoGroup Map API.
+  repeated Expression input_sorting_expressions = 6;
```

Comment:
I think this is better to named `left_input_sorting_expressions` and another is `right_input_sorting_expressions`?

Reply:
Currently we follow the names used in the proto. The sorting expressions are one for the input and one for the other input.

Reply:
@xinrong-meng why did you name it `input` and `other`, instead of `left` and `right`?

Reply:
When there is only one relation involved, we name it `input` by convention. Thus, an additional relation is named `other`.  I don't have an objection if you prefer renaming. I would suggest minimizing changes to existing proto though.